internal/pkg/ai-functions/handler/handler_ai_function.go

```diff
@@ -51,16 +54,86 @@ func (h *AIFunction) Apply(ctx context.Context, req *pb.ApplyRequest) (pbValue *
 		err := errors.Errorf("AI function %s not found", req.GetFunctionName())
 		return nil, HTTPError(err, http.StatusBadRequest)
 	}
+
+	var results any
+	switch req.GetFunctionName() {
```

Comment:
abstract these logic, for multiple scenarios in the future.

Reply:
OK